data/remnant/remnant ships.txt

```diff
@@ -1183,33 +1170,30 @@ ship "Peregrine"
 		licenses
 			"Remnant Capital"
 		"cost" 41627000
-		"shields" 16400
-		"hull" 10600
+		"shields" 15400
+		"hull" 8600
 		"required crew" 9
 		"bunks" 25
 		"mass" 350
 		"drag" 4.3
 		"heat dissipation" 0.85
-		"fuel capacity" 1100
+		"fuel capacity" 900
 		"ramscoop" 3.25
-		"cargo space" 59
+		"cargo space" 49
 		"outfit space" 552
 		"weapon capacity" 220
 		"engine capacity" 171
-		"shield generation" 4.9
+		"shield generation" 3.9
 		"shield energy" 3.0
-		"hull repair rate" 2.5
+		"hull repair rate" 1.5
 		"hull energy" 0.90
-		"cloak" .04
-		"cloaking energy" 2.5
+		"cloak" .03
+		"cloaking energy" 3
 		"cloaking fuel" .05
 		"gaslining" 1
 		"outfit scan power" 18
 		"outfit scan efficiency" 22
-		"tactical scan power" 50
-		"ion protection" 0.25
-		"scramble protection" 0.25
-		"slowing protection" 0.25
+		"tactical scan power" 39
```

Comment:
Just across the board: no. The Peregrine already doesn't stand up well in late game combat; it's a medium warship, and it shows. Its only saving grace is the built-in cloak, which you are nerfing. None of these changes are necessary. Some are completely inexplicable, like the bizarre removals of integrated ion/scramble/slowing protection. Others just seem petty, like the arbitrary decrease of cargo space.

I also see you already had to adjust a mission just to be able to accommodate the decreased cargo space, by the way. For what?

I recognize that the ship was buffed from its original version, but that by itself does not give justification for reverting it for the sake of reverting it.

Reply:
Cloaking is still really powerful to the point where balancing checkmate/reconciliation is a major issue and an overwhelming majority of players choose reconciliation because of it.

Reply:
Are you arguing that the people who reviewed and merged https://github.com/endless-sky/endless-sky/pull/6392 just somehow forgot to pay attention to the changes made to the Peregrine? Or are you just commenting on the fact that nobody directly commented on those specific lines?

Since historical context is apparently necessary, I'll add that Peregrine access was originally granted in that PR, so bringing up the unused Peregrine stat block is weird. Moreover, the Peregrine is now a unique ship, with no more cappable nor purchasable Peregrines existing in the game. 

This nerf is not reasonable, but even if it were reasonable, it would merit actual justification, not what amounts to "somebody changed it and I didn't notice".

Reply:
It was not an unrelated PR. #6392 is the PR which _made the Peregrine available_, and that's noted in the PR text. The fact that the changed stats somehow escaped your knowledge doesn't by itself justify a revert.

Reply:
Which reviewer? What did they say exactly? I'm sure they can speak for themselves here...

Reply:
> They didn't escape my attention, they escaped the attention _of a reviewer who asked me to include the reversion_

They escaped the attention of _everyone_, as far as I can tell looking through the comments on the PR. I wasn't a reviewer at the time, so maybe that statement is slightly misleading.
My reasoning for pointing this out to Azure was that they have been discussing the fact that the Peregrine (and Merganser) have an unreasonable amount of innate capabilities, without the reduced outfit space to offset that and balance out overall that other Remnant ships, such as the Starling, Gull, Ibis and Albatross have.
I noticed the change when I was looking for the PR that added the Peregrine, so I could look at all the discussion on it and see what had been said already.
I'm not committed to the reversion of the change on principle, but I feel it's a reasonable starting point for proposing a nerf to the Peregrine. Since this discussion is now being held openly and with proper input, there's no reason we need necessarily to stick to this specific set of changes if the discussion leads to a consensus on other changes, but this is a sensible starting point.

Reply:
Yes, it seems that this was a miscommunication. I couldn't see any mention of the changes from anyone on the PR, so I reasoned that they had been missed, but it seems that wasn't the case. Feel free to explain how I was meant to intuit from a total absence of any mention of the change that people were aware of it and approved of it, and the precise manner in which what I thought had happened would have looked different.

So, that grounds for this specific set of changes over any other isn't there. But, [as I already pointed out](https://github.com/endless-sky/endless-sky/pull/9693#issuecomment-1902784300), the Peregrine has some _huge_ advantages (cloaking with 1/4 the fuel cost of the cloaking device, nearly 200 tons worth of integrated capabilities on top of that, extremely high top speed) over comparable ships, with no real disadvantages to offset them. That's the justification for why stats need to be reduced. There is scope for discussion over the exact values, as there would have been before.

Reply:
Hey gang! Wanted to provide some context on the ship balancing changes made in #6392 since I reviewed and merged it. I've looked over my own comments and everything else I have related to my review of that PR (pretty much just DMs). Generally, balancing is not my area of specialty; I try to request other balance-minded folks check those sections of PRs I review, and I trust their approval of those sections. Honestly, I think I just forgot to do that for this one, especially since this was the first large PR I merged.
As far as I know, I didn't comment on or request any ship balancing changes on #6392. That's on me for not being thorough enough, so my apologies. So if there is community consensus that the content I approved is unbalanced, I'm inclined to think they're probably right given my own lack of experience in balancing. So in this case, I think my approval and merging of the PR doesn't exempt the balancing changes from further deliberation.
Again, my bad for missing those changes. Also, I might not be the only one who was involved in the balancing conversation, so I can only cover things from my perspective. Hope this helps!